app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -0,0 +1,107 @@
+import styled, { css } from "styled-components";
+
+const H1 = styled.h1`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 28px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
+`;
+
+const H2 = styled.h2`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 24px;
+  font-style: normal;
+  line-height: 1.2;
+  text-transform: capitalize;
+  color: #3e4449;
+`;
+
+const H3 = styled.h3`
+  font-family: "Roboto";
+  font-weight: 400;
+  font-size: 18px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
+`;
+
+const H3bold = styled.h3`
+  font-family: "Roboto";
+  font-weight: 600;
+  font-size: 18px;
+  font-style: normal;
+  line-height: 1.2;
+  color: #3e4449;
+`;
```

Comment:
Here we can have just one component with a prop:
```suggestion
const H3 = styled.h3<{bold?: boolean}>`
  font-weight: ${(p) => p.bold ? '600' : '400'};
  font-size: 18px;
`;
```

I've written it here, but most probably should work.